client/Cargo.toml

```diff
@@ -22,7 +22,7 @@ client-sync = ["jsonrpc"]
 
 [dependencies]
 bitcoin = { version = "0.32.0", default-features = false, features = ["std", "serde"] }
-types = { package = "corepc-types", version = "0.9.0", default-features = false, features = ["std"] }
+types = { package = "corepc-types", path = "../types", default-features = false, features = ["std"] }
```

Comment:
Can you change the deps for all crates in this repo to use `path` instead of `version` please. Its superior in all ways AFAICT, I just didn't know that when I started this repo.